menas/ui/components/dataset/conformanceRule/ConformanceRuleForm.js

```diff
@@ -386,9 +395,18 @@ class MappingConformanceRuleForm extends ConformanceRuleForm {
     return isValid
   }
 
+  hasValidOutputColumns(rule = []) {
+    let isValid = rule.additionalColumns.length >= 1 || rule.outputColumn !== undefined;
+
```

Comment:
I think the check here is less strict for the `additionalColumns` then in `hasValidOutputColumn()`

Reply:
True, because `hasValidOutputColumn` cannot be applied here. `SchemaManager.validatePathOfStructs` is applied in `OutputColumnDialog`